### ISSUE TITLE
.net 8 > Could not load file or assembly 'Accessibility, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'. The system cannot find the file specified.

### ISSUE THREAD

**Issue by AsheeshGareeb** [FlaUI/FlaUI#684]
i am new to Flaui, can anyone help me regarding an urgent help

I migrated my system from .net 4.7 to .net8 now i am getting error on below line

_automation = new UIA3Automation();

System.IO.FileNotFoundException
HResult=0x80070002
Message=Could not load file or assembly 'Accessibility, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'. The system cannot find the file specified.
Source=FlaUI.UIA3
StackTrace:
at FlaUI.UIA3.UIA3PatternLibrary.get_LegacyIAccessiblePattern()
at FlaUI.UIA3.UIA3PatternLibrary.get_AllForCurrentFramework()
at FlaUI.Core.AutomationBase..ctor(IPropertyLibrary propertyLibrary, IEventLibrary eventLibrary, IPatternLibrary patternLibrary, ITextAttributeLibrary textAttributeLibrary)
at FlaUI.UIA3.UIA3Automation..ctor()
at CiqproOffice.StepDefinitions.SmokeTestStepDefinitions.GivenILauchExcel() in C:\Users\agareeb\source\repos\CiqproOffice\StepDefinitions\SmokeTestStepDefinitions.cs:line 45
at System.RuntimeMethodHandle.InvokeMethod(Object target, Void** arguments, Signature sig, Boolean isConstructor)
at System.Reflection.MethodBaseInvoker.InvokeDirectByRefWithFewArgs(Object obj, Span`1 copyOfArgs, BindingFlags invokeAttr)

**Comment by rikumaru611** [comment]
@AsheeshGareeb 

Hi. AsheeshGareeb.
I face to same issue.
In my case, installed FlaUI.Core and FlaUI and FlaUI.UIA3 **version 4.0.0**, And update version 5.0.0 resolve the exception.
I consider the FlaUI.UIA3 version 5.0.0 miss installing the Interop.UIAutomationClient, and it is cause of the exception.
Therefore, I needed to install FlaUI,UI3 version 4.0.0 at first and update it.

I hope this helps, and I wish you success!

**Comment by tomczakmarcel** [comment]
Hi. 

For my .NET9 Nunit test project problem was resolved when I've added this to my csproj:

	  <TargetFramework>net9.0-windows</TargetFramework>

**Comment by Jun-GY** [comment]
I also encountered this problem. The project is a console program and the target framework is .Net8

![Image](https://github.com/user-attachments/assets/8976a304-a95c-4546-b48a-bf2adf5c912d)

**Comment by Jun-GY** [comment]
This problem can be solved by including the accessibility package.

![Image](https://github.com/user-attachments/assets/3226bbdd-a669-4d8a-bbbe-ea816d5e1fbe)

**Comment by AsheeshGareeb** [comment]
it worked by changing 
<TargetFramework>net9.0</TargetFramework>
to
<TargetFramework>net9.0-windows</TargetFramework>

**Comment by AsheeshGareeb** [comment]
Thank you everyone for the response, i am closing this thread!!
